openslides_backend/action/meeting/delete_all_speakers_of_all_lists.py

```diff
@@ -15,7 +15,11 @@ class DeleteAllSpeakersOfAllListsAction(DeleteAction):
     """
 
     model = Speaker()  # we want to delete Speakers
-    schema = DefaultSchema(Speaker()).get_delete_schema()
+    schema = DefaultSchema(Speaker()).get_default_schema(
```

Comment:
Although it makes no difference in this case, it is still not quite correct, since the base model for the schema is `meeting`. Strictly speaking, you are accesing the property `speaker.id` here, but want to have `meeting.id`. Of course, this is the same schema, but if for some reason other fields are added, it would produce wrong results. So please change it to `Meeting()`.

Reply:
Updated. Now I use Meeting instead Speaker in DefaultSchema.